tlair/src/main/scala/at/forsyte/apalache/tla/typecomp/unsafe/UnsafeSetBuilder.scala

```diff
@@ -87,10 +87,14 @@ class UnsafeSetBuilder extends ProtoBuilder {
     // Even, non-zero number of args and every other argument is NameEx
     require(TlaSetOper.map.arity.cond(1 + pairs.size), s"Expected pairs to have even, positive arity, found $pairs.")
     val (vars, _) = TlaOper.deinterleave(pairs)
-    require(vars.forall { _.isInstanceOf[NameEx] }, s"Expected vars to be variable names, found $vars.")
+    val varsMultiseq = vars.flatMap(BuilderUtil.getBoundVarsOrThrow) // any can throw if not the right form
```

Comment:
What does `any` mean here in the comment?

Reply:
"Any of the tuple components"